ioctl/newcmd/contract/contractcompile_test.go

```diff
@@ -19,6 +19,7 @@ import (
 )
 
 func TestNewContractCompileCmd(t *testing.T) {
+	t.Skip("Skip test due to solc correct version not installed in CI")
```

Comment:
file an issue for this?

Reply:
ok, removed and the new issue filed https://github.com/iotexproject/iotex-core/issues/3816